quality_check/09-prior_information_matrix.Rmd

```diff
@@ -0,0 +1,225 @@
+---
+title: "Construct _Danio rerio_ prior information matrices"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: J. Taroni for CCDL
+date: 2019
+---
+
+If we want to use `PLIER`, we need to construct prior information matrices
+for zebrafish.
+We'll do that using two sources: 
+[WikiPathways](https://www.wikipathways.org/index.php/WikiPathways) and 
```

Comment:
Why WikiPathways as opposed to other sources for biological pathways? I'm not asking this to imply we shouldn't use wikipathways, but just wondering what you like about this one because I don't know much about it.

Reply:
I could easily obtain a `.gmt` file for zebrafish and GO terms do not seem like a particularly good fit for use with PLIER in my opinion. In the future, I will most likely want to repeat this with KEGG pathways but it is not immediately obvious to me how to get that information.